### DISCUSSION TITLE
[Bug] Failed to run Kyuubi Server on K8S

### DISCUSSION THREAD

**Discussion by zhaohehuhu** [apache/kyuubi#3434] [Q&A] [answered]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Describe the bug

When I was ready to run Kyuubi Servers on our K8S cluster as a deployment, I came across a problem below.  The reason is the Kyuubi user failed to perform application because of permission.

### Affects Version(s)

master

### Kyuubi Server Log Output

```logtalk
starting container process caused: exec: "./bin/kyuubi": stat ./bin/kyuubi: permission denied: unknown
```

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

**Comment by pan3793** [comment]
How do you build your docker image?

**Reply by zhaohehuhu** [reply]
just run the command(docker build --tag xx --file build/Dockerfile .)

**Reply by zhaohehuhu** [reply]
My workaround is granting execute permission  for Kyuubi user. It's ok?

**Comment by yaooqinn** [comment]
this is not a proper way to define a bug without any description, I will move it to discussions

**Reply by zhaohehuhu** [reply]
OK.Thanks.

**Comment by yaooqinn** [comment]
cc @zwangsheng, is this a bug indeed, or just an issue of use environment?

**Comment by zwangsheng** [comment, marked as answer]
Hello, kyuubi provide two Dockerfiles in project, `./build/Dockerfile` for Github CI/CD and `./docker/Dockerfile` for users build custom docker image.
We recommend use `./docker/Dockerfile` and `./bin/docker-image-tool.sh` to build your docker image.
See more detail about this in https://kyuubi.readthedocs.io/en/latest/deployment/kyuubi_on_kubernetes.html#build-kyuubi-docker-image

**Reply by zhaohehuhu** [reply]
I used to run the command(docker build --tag xx --file build/Dockerfile .). I will try your solution to see if there is same issue. Thanks.
